drainer/metrics.go

```diff
@@ -74,6 +74,14 @@ var (
 			Help:      "save checkpoint tso of drainer.",
 		})
 
+	checkpointDelayGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: "binlog",
+			Subsystem: "drainer",
+			Name:      "checkpoint_delay",
```

Comment:
better let people know the unit, like name `check_delay_seconds`

Reply:
Fixed